src/components/Badge/__stories__/Badge.stories.mdx

```diff
@@ -0,0 +1,224 @@
+import Badge from "../Badge";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createStoryMetaSettings } from "../../../storybook";
+import styles from "./Badge.stories.module.scss";
+import person from "./assets/person.png";
+import { ExternalPage, WhatsNew } from "../../Icon/Icons";
+import Link from "../../Link/Link";
+import Counter from "../../Counter/Counter";
+import Button from "../../Button/Button";
+import Flex from "../../Flex/Flex";
+import Avatar from "../../Avatar/Avatar";
+import { useState } from "react";
+import Indicator from "../../Indicator/Indicator";
+
+export const metaSettings = createStoryMetaSettings({
+  component: Badge,
+  enumPropNamesArray: ["position"], // List enum props here
+  iconPropNamesArray: [], // List props that are typed as icons here
+  actionPropsArray: [] // List the component's actions here
+});
+
+<Meta
+  title="Components/Badge/Badge"
+  component={Badge}
+  argTypes={metaSettings.argTypes}
+  decorators={metaSettings.decorators}
+/>
+
+<!--- Component documentation -->
+
+# Badge
+
+- [Overview](#overview)
+- [Props](#props)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+
+Badge component is responsible for positioning an indicator/badge on a child component
+
+<Canvas>
+  <Story name="Overview">
+    <Badge>
+      <Button leftIcon={WhatsNew}>What's new</Button>
```

Comment:
Overview story should be written in a way users will be able to use controls panel, see how we did it in other components :)